### DISCUSSION TITLE
Trasformare i task che emergono dalle riunioni in issue?

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/informa-covid19#3] [Idee/Proposte]
Ciao,
che ne pensate di trasformarle in issue, sfruttando tutte le comodità di gestione, avvisi, lavori di gruppo, log?

Come ad esempio quella su "[tabella fonte dati comune](https://groups.google.com/g/opendatasicilia/c/OM82Zc-PKII/m/nhoeEBiCAgAJ)"

**Comment by gpirrotta** [comment]
Buona idea!
